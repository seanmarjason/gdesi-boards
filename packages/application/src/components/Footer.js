import * as React from 'react';
import Box from '@mui/material/Box';
import Container from '@mui/material/Container';
import Stack from '@mui/material/Stack';
import GdesiIcon from './GdesiIcon';

export default function Footer() {
  return (
    <Container
      sx={{
        display: 'flex',
        flexDirection: 'column',
        alignItems: 'center',
        gap: { xs: 4, sm: 8 },
        py: { xs: 8, sm: 10 },
        textAlign: { sm: 'center', md: 'left' },
      }}
    >
      <Box
        sx={{
          display: 'flex',
          flexDirection: { xs: 'column', sm: 'row' },
          width: '100%',
          justifyContent: 'space-between',
        }}
      >
        <Box
          sx={{
            display: 'flex',
            flexDirection: 'column',
            gap: 4,
            minWidth: { xs: '100%', sm: '60%' },
          }}
        >
          <Box sx={{ width: { xs: '100%', sm: '60%' } }}>
            <GdesiIcon />
            <Stack direction="row" spacing={1} useFlexGap>

            </Stack>
          </Box>
        </Box>
        <Box
          sx={{
            display: { xs: 'none', sm: 'flex' },
            flexDirection: 'column',
            gap: 1,
          }}
        >
        </Box>
        <Box
          sx={{
            display: { xs: 'none', sm: 'flex' },
            flexDirection: 'column',
            gap: 1,
          }}
        >
        </Box>
        <Box
          sx={{
            display: { xs: 'none', sm: 'flex' },
            flexDirection: 'column',
            gap: 1,
          }}
        >
        </Box>
      </Box>
      <Box
        sx={{
          display: 'flex',
          justifyContent: 'space-between',
          pt: { xs: 4, sm: 8 },
          width: '100%',
          borderTop: '1px solid',
          borderColor: 'divider',
        }}
      >
        <div>
        </div>
        <Stack
          direction="row"
          spacing={1}
          useFlexGap
          sx={{ justifyContent: 'left', color: 'text.secondary' }}
        >
        </Stack>
      </Box>
    </Container>
  );
}
