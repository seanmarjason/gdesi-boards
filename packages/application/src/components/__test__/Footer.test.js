import { render } from '@testing-library/react'
import Footer from '../Footer';

it('Footer renders correctly', () => {
  const { container } = render(<Footer />)
  expect(container).toMatchSnapshot()
})
