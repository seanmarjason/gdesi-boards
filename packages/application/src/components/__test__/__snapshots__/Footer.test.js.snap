// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-17h02ml-MuiContainer-root"
  >
    <div
      class="MuiBox-root css-fdshit"
    >
      <div
        class="MuiBox-root css-5dev9e"
      >
        <div
          class="MuiBox-root css-unvpgn"
        >
          <img
            alt="The house from the offer."
            class="MuiBox-root css-ndi7rl"
            src="desi-sm-to2.png"
          />
          <p
            class="MuiTypography-root MuiTypography-body2 MuiTypography-gutterBottom css-dx2fqb-MuiTypography-root"
          >
            Join the newsletter
          </p>
          <p
            class="MuiTypography-root MuiTypography-body2 css-19b5ozw-MuiTypography-root"
          >
            Subscribe for weekly updates. No spams ever!
          </p>
          <label
            class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-animated css-1tkmop-MuiFormLabel-root-MuiInputLabel-root"
            for="email-newsletter"
          >
            Email
          </label>
          <div
            class="MuiStack-root css-1oxswqj-MuiStack-root"
          >
            <div
              aria-label="Enter your email address"
              class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-yzzevn-MuiFormControl-root-MuiTextField-root"
            >
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-sizeSmall MuiInputBase-hiddenLabel css-1blp12k-MuiInputBase-root-MuiOutlinedInput-root"
              >
                <input
                  aria-invalid="false"
                  aria-label="Enter your email address"
                  autocomplete="off"
                  class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSizeSmall MuiInputBase-inputHiddenLabel css-1pzfmz2-MuiInputBase-input-MuiOutlinedInput-input"
                  id="email-newsletter"
                  placeholder="Your email address"
                  type="text"
                />
                <fieldset
                  aria-hidden="true"
                  class="MuiOutlinedInput-notchedOutline css-1ll44ll-MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="css-w4cd9x"
                  >
                    <span
                      aria-hidden="true"
                      class="notranslate"
                    >
                      ​
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall MuiButton-containedSizeSmall MuiButton-colorPrimary MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall MuiButton-containedSizeSmall MuiButton-colorPrimary css-1at5l3r-MuiButtonBase-root-MuiButton-root"
              id=":r1:"
              tabindex="0"
              type="button"
            >
              Subscribe
            </button>
          </div>
        </div>
      </div>
      <div
        class="MuiBox-root css-yvp7c"
      >
        <p
          class="MuiTypography-root MuiTypography-body2 css-1n8v0zg-MuiTypography-root"
        >
          Product
        </p>
        <a
          class="MuiTypography-root MuiTypography-body2 MuiLink-root MuiLink-underlineAlways css-1dzosyc-MuiTypography-root-MuiLink-root"
          href="#"
          style="--Link-underlineColor: rgba(0, 0, 0, 0.4);"
        >
          Features
        </a>
        <a
          class="MuiTypography-root MuiTypography-body2 MuiLink-root MuiLink-underlineAlways css-1dzosyc-MuiTypography-root-MuiLink-root"
          href="#"
          style="--Link-underlineColor: rgba(0, 0, 0, 0.4);"
        >
          FAQs
        </a>
      </div>
      <div
        class="MuiBox-root css-yvp7c"
      >
        <p
          class="MuiTypography-root MuiTypography-body2 css-1n8v0zg-MuiTypography-root"
        >
          Company
        </p>
        <a
          class="MuiTypography-root MuiTypography-body2 MuiLink-root MuiLink-underlineAlways css-1dzosyc-MuiTypography-root-MuiLink-root"
          href="#"
          style="--Link-underlineColor: rgba(0, 0, 0, 0.4);"
        >
          About us
        </a>
        <a
          class="MuiTypography-root MuiTypography-body2 MuiLink-root MuiLink-underlineAlways css-1dzosyc-MuiTypography-root-MuiLink-root"
          href="#"
          style="--Link-underlineColor: rgba(0, 0, 0, 0.4);"
        >
          Careers
        </a>
        <a
          class="MuiTypography-root MuiTypography-body2 MuiLink-root MuiLink-underlineAlways css-1dzosyc-MuiTypography-root-MuiLink-root"
          href="#"
          style="--Link-underlineColor: rgba(0, 0, 0, 0.4);"
        >
          Press
        </a>
      </div>
      <div
        class="MuiBox-root css-yvp7c"
      >
        <p
          class="MuiTypography-root MuiTypography-body2 css-1n8v0zg-MuiTypography-root"
        >
          Legal
        </p>
        <a
          class="MuiTypography-root MuiTypography-body2 MuiLink-root MuiLink-underlineAlways css-1dzosyc-MuiTypography-root-MuiLink-root"
          href="#"
          style="--Link-underlineColor: rgba(0, 0, 0, 0.4);"
        >
          Terms
        </a>
        <a
          class="MuiTypography-root MuiTypography-body2 MuiLink-root MuiLink-underlineAlways css-1dzosyc-MuiTypography-root-MuiLink-root"
          href="#"
          style="--Link-underlineColor: rgba(0, 0, 0, 0.4);"
        >
          Privacy
        </a>
        <a
          class="MuiTypography-root MuiTypography-body2 MuiLink-root MuiLink-underlineAlways css-1dzosyc-MuiTypography-root-MuiLink-root"
          href="#"
          style="--Link-underlineColor: rgba(0, 0, 0, 0.4);"
        >
          Contact
        </a>
      </div>
    </div>
    <div
      class="MuiBox-root css-9or4r7"
    >
      <div>
        <a
          class="MuiTypography-root MuiTypography-body2 MuiLink-root MuiLink-underlineAlways css-1dzosyc-MuiTypography-root-MuiLink-root"
          href="#"
          style="--Link-underlineColor: rgba(0, 0, 0, 0.4);"
        >
          Privacy Policy
        </a>
        <p
          class="MuiTypography-root MuiTypography-body1 css-h43rkw-MuiTypography-root"
        >
           • 
        </p>
        <a
          class="MuiTypography-root MuiTypography-body2 MuiLink-root MuiLink-underlineAlways css-1dzosyc-MuiTypography-root-MuiLink-root"
          href="#"
          style="--Link-underlineColor: rgba(0, 0, 0, 0.4);"
        >
          Terms of Service
        </a>
        <p
          class="MuiTypography-root MuiTypography-body2 css-tmy7vb-MuiTypography-root"
        >
          Copyright © 
          <a
            class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1yd3lbv-MuiTypography-root-MuiLink-root"
            href="https://mui.com/"
            style="--Link-underlineColor: rgba(0, 0, 0, 0.4);"
          >
            Gdesi
          </a>
           
          2025
        </p>
      </div>
      <div
        class="MuiStack-root css-mq9tmk-MuiStack-root"
      >
        <a
          aria-label="GitHub"
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeSmall css-pzyrwo-MuiButtonBase-root-MuiIconButton-root"
          href="https://github.com/mui"
          id=":r2:"
          tabindex="0"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
            data-testid="GitHubIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 1.27a11 11 0 00-3.48 21.46c.55.09.73-.28.73-.55v-1.84c-3.03.64-3.67-1.46-3.67-1.46-.55-1.29-1.28-1.65-1.28-1.65-.92-.65.1-.65.1-.65 1.1 0 1.73 1.1 1.73 1.1.92 1.65 2.57 1.2 3.21.92a2 2 0 01.64-1.47c-2.47-.27-5.04-1.19-5.04-5.5 0-1.1.46-2.1 1.2-2.84a3.76 3.76 0 010-2.93s.91-.28 3.11 1.1c1.8-.49 3.7-.49 5.5 0 2.1-1.38 3.02-1.1 3.02-1.1a3.76 3.76 0 010 2.93c.83.74 1.2 1.74 1.2 2.94 0 4.21-2.57 5.13-5.04 5.4.45.37.82.92.82 2.02v3.03c0 .27.1.64.73.55A11 11 0 0012 1.27"
            />
          </svg>
        </a>
        <a
          aria-label="X"
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeSmall css-pzyrwo-MuiButtonBase-root-MuiIconButton-root"
          href="https://x.com/MaterialUI"
          id=":r3:"
          tabindex="0"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
            data-testid="XIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M18.244 2.25h3.308l-7.227 8.26 8.502 11.24H16.17l-5.214-6.817L4.99 21.75H1.68l7.73-8.835L1.254 2.25H8.08l4.713 6.231zm-1.161 17.52h1.833L7.084 4.126H5.117z"
            />
          </svg>
        </a>
        <a
          aria-label="LinkedIn"
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeSmall css-pzyrwo-MuiButtonBase-root-MuiIconButton-root"
          href="https://www.linkedin.com/company/mui/"
          id=":r4:"
          tabindex="0"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
            data-testid="LinkedInIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
            />
          </svg>
        </a>
      </div>
    </div>
  </div>
</div>
`;
